installer.sh

```diff
@@ -496,6 +584,9 @@ echo "==========================================================================
 echo $'\t\t\t\tInstall Keylime'
 echo "=================================================================================="
 cd $KEYLIME_DIR
+if [[ "$NEED_PYTHON_DIR" -eq "1" ]] ; then
```

Comment:
Is this really needed?  It seems something like this should be handled by something else ...

Reply:
on my centos 8 box, keylime wouldn't install correctly without this.  I think @lukehinds told me about it?

Reply:
This could explain it: https://github.com/pypa/setuptools/pull/1431#issuecomment-406640252

Reply:
i'm not sure what to do about this since the installer runs as root.  Hopefully it will just go away once we can package for centos 8.

Reply:
The majority of things in this script don't even need sudo privileges, so we could probably have it run without it and just sudo the commands that actually need root.  The downside here is that if a user has disabled sudo cache then they'll have to type their password 20 times.